suite-common/wallet-core/src/accounts/accountsThunks.ts

```diff
@@ -20,24 +20,38 @@ import { notificationsActions } from '@suite-common/toast-notifications';
 import { transactionsActions } from '../transactions/transactionsActions';
 import { selectTransactions } from '../transactions/transactionsReducer';
 import { accountsActions } from './accountsActions';
-import { selectAccountByKey, selectAccounts } from './accountsReducer';
+import {
+    selectAccountByKey,
+    selectAccounts,
+    selectPortfolioTrackerAccounts,
+} from './accountsReducer';
 import { ACCOUNTS_MODULE_PREFIX } from './accountsConstants';
 import { selectBlockchainHeightBySymbol } from '../blockchain/blockchainReducer';
 
 export const disableAccountsThunk = createThunk(
     `${ACCOUNTS_MODULE_PREFIX}/disableAccountsThunk`,
-    (_, { dispatch, extra, getState }) => {
+    (
+        { excludePortfolioTrackerAccounts = false }: { excludePortfolioTrackerAccounts?: boolean },
```

Comment:
`{ excludePortfolioTrackerAccounts = false }: { excludePortfolioTrackerAccounts?: boolean } | undefined = {},`

little akward but then you shoudn't need to put `{}` to every call